userspace/engine/falco_engine_version.h

```diff
@@ -24,4 +24,4 @@ limitations under the License.
 // This is the result of running "falco --list -N | sha256sum" and
 // represents the fields supported by this version of falco. It's used
 // at build time to detect a changed set of fields.
-#define FALCO_FIELDS_CHECKSUM "9b5557ec8f16f5606a1544573b152d211d5212f653ee039146836a17266ff449"
+#define FALCO_FIELDS_CHECKSUM "ceb069d9f9b2d4ebcc5de39bddc53b7af2e6b8f072edc293668fd6ac4e532413"
```

Comment:
When you update the checksum, that almost always means new fields were added to either sysdig (for syscall events) or falco (for k8s audit events). When that happens, you should also bump the `FALCO_ENGINE_VERSION` define on line 22. That allows us to know the times at which the engine was upgraded to have new fields. When you actually *use* those fields, you should update the rules file that uses them to have a `- required_engine_version: NNN` where NNN is the current value of `FALCO_ENGINE_VERSION`.

Reply:
good to know, is this documented anywhere or should we do it?

Reply:
From a user perspective, there are some notes at https://falco.org/docs/rules/#versioning. Not much for developer specific notes, though.